src/types/duration.rs

```diff
@@ -0,0 +1,59 @@
+use napi::{bindgen_prelude::BigInt, Error, Status};
+use scylla::frame::value::CqlDuration;
+
+#[napi]
+pub struct DurationWrapper {
+    pub months: i32,
+    pub days: i32,
+    pub nanoseconds: i64,
+}
+
+#[napi]
+impl DurationWrapper {
+    #[napi]
+    pub fn new(months: i32, days: i32, ns_bigint: BigInt) -> napi::Result<Self> {
+        let (ns_value, is_lossless) = ns_bigint.get_i64();
+        if !is_lossless {
+            return Err(Error::new(
+                Status::GenericFailure,
+                "Nanoseconds must not overflow i64",
+            ));
+        }
```

Comment:
Now I'm scared when seeing this `BigInt::get_i64()`...

Reply:
Are we going to replace this with the new util function from #85?

Reply:
I plan to merge #85, update this PR to use those functions and then merge this PR